### PR TITLE
iframe load issue

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/services/visualizations/iframe.service.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
     function GenerateIframe(VisualizationUtils, $rootScope, D3Utils, filterParametersService) {
         return {
             build: function (record, element, panel, isNotification) {
-                var iframeLink = VisualizationUtils.getPropertyValue(record.properties, 'I frame link');
+                var iframeLink = VisualizationUtils.getPropertyValue(record.properties, 'iFrame link');
```
Comment (johibkhan2) [inline src/main/webapp/app/services/visualizations/iframe.service.js:13]: Can you put it in constant file iFrame link

Comment (khushbu-maurya) [reply src/main/webapp/app/services/visualizations/iframe.service.js:13]: its come from database

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**